db/migrate/20200109094012_add_integrity_check_problems_to_whitehall_document_migration.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddIntegrityCheckProblemsToWhitehallDocumentMigration < ActiveRecord::Migration[6.0]
+  def change
+    change_table :whitehall_migration_document_imports, bulk: true do |t|
+      t.column :integrity_check_problems, :string, array: true
+      t.column :integrity_check_proposed_payload, :json, default: "{}"
```

Comment:
Should this be null: false if we're setting it as a default empty object? Similar thoughts on the problems column should that have a default of empty and null false

Minor but it seems unusual that these have t.column rather than t.json so if we edit these lines it'd be nice to make them more consistent with other migrations

Reply:
I've changed these to `t.string` and `t.json` and set `null: false`